manage_breast_screening/participants/models/appointment.py

```diff
@@ -74,6 +74,17 @@ def order_by_starts_at(self, desc=False):
             "-clinic_slot__starts_at" if desc else "clinic_slot__starts_at"
         )
 
+    def eager_load_current_status(self):
```

Comment:
maybe this should be named prefetch_current_status to use the same terminology as django?